src/containers/shared/components/Transaction/XChainClaim/test/XChainClaimSimple.test.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import { I18nextProvider } from 'react-i18next'
+import { BrowserRouter } from 'react-router-dom'
+import { mount, ReactWrapper } from 'enzyme'
+import { Simple } from '../Simple'
+import mockXChainClaim from './mock_data/XChainClaim.json'
+import summarizeTransaction from '../../../../../../rippled/lib/txSummary'
+import i18n from '../../../../../../i18nTestConfig'
+
+function createWrapper(tx: any) {
+  const data = summarizeTransaction(tx, true)
+  return mount(
+    <I18nextProvider i18n={i18n}>
+      <BrowserRouter>
+        <Simple data={data.details} />
+      </BrowserRouter>
+    </I18nextProvider>,
+  )
+}
+
+function expectText(
+  wrapper: ReactWrapper<any, Readonly<{}>>,
+  dataTest: string,
+  text: string,
+) {
+  expect(wrapper.find(`[data-test="${dataTest}"] .value`)).toHaveText(text)
```

Comment:
Why is there a space between `[data-test="${dataTest}"]` and `.value`?

Reply:
That's just how the `wrapper.find` method works - maybe @ckniffen has more details on that

Reply:
It is a css selector which means find the element(s) with the  attribute `data-test` set to something and then find the element(s) with the class name of `value`